src/main/java/com/powsybl/balances_adjustment/balance_computation/BalanceComputationParameters.java

```diff
@@ -36,21 +37,24 @@
      */
     private int maxNumberIterations;
 
+    private boolean constantPowerScaling;
```

Comment:
Can you please use exactly the same parameter as those used in the open loadflow? See the documentation https://www.powsybl.org/pages/documentation/simulation/powerflow/openlf.html for more details.

Reply:
@annetill Maybe it is relevant to put this parameter in `LoadFlowParameters`? When a user wants to do a constant power scaling, they will want to have both of the parameters set to true, won't they? If it is the case, maybe only one parameter in `LoadFlowParameters` is enough: it prevents error since `BalanceComputationParameters` contain a `LoadflowParameters` object.

Reply:
I am not sure about that because for me you have the right to:
- Run a power flow with distributed slack on generation ;
- Scale loads in the ajustement with a power factor constant.